docs/WindowsBuild.md

```diff
@@ -89,6 +89,12 @@ set swift_source_dir=path-to-directory-containing-all-cloned-repositories
   (`Debug`, `RelWithDebInfoAssert` or `Release`) to avoid conflicts between the debug and 
   non-debug version of the MSCRT library.
 
+- Set up the `visualc` and `ucrt` modules by copying  `ucrt.modulemap` located at
```

Comment:
Copy the text that defines UniversalCRTSdkDir etc into this file, too.

Reply:
All three of the variables should be defined automatically in the Developer Command Prompt.